### DISCUSSION TITLE
Is it possible to produce a webp or animated GIF output instead of LED output

### DISCUSSION THREAD

**Discussion by jdesai61** [MLB-LED-Scoreboard/mlb-led-scoreboard#514] [Q&A] [answered]
Using the emulator option, is it possible to output to an animated GIF or webp file instead? Idea is to then serve the output to tidbyt (or like device) over HTTP. Thanks

**Comment by WardBrian** [comment, marked as answer]
Probably better asked in the emulator repo: https://github.com/ty-porter/RGBMatrixEmulator

**Reply by ty-porter** [reply]
👍🏻 would be happy to have a discussion there.
